src/resources/UsageAnalytics/Read/Users/UsersInterfaces.ts

```diff
@@ -0,0 +1,121 @@
+export interface UsersFilterModel {
+    /**
+     * The filter id
+     */
+    id: string;
+    /**
+     * The filter display name
+     */
+    displayName: string;
+    /**
+     * The filter account
+     */
+    account: string;
+    /**
+     * The filter type. Can be 'permissions' or 'reporting'
+     */
+    type: string;
+    /**
+     * The actual filter
+     */
+    value: string;
+}
+
+export interface UsersFiltersModel {
+    /**
+     * The filters that are applied to the user
+     */
+    filters: UsersFilterModel[];
+}
+
+export interface UsersReportModel {
+    /**
+     * The report id
+     */
+    id: string;
+    /**
+     * The report account
+     */
+    account: string;
+    /**
+     * The report display name
+     */
+    displayName: string;
+    /**
+     * The report type. Can be 'permissions' or 'reporting'
+     */
+    type: string;
+    /**
+     * The report configuration in JSON
+     */
+    configuration: Record<string, any>;
```

Comment:
In an ideal world 🌈 I'd like to have an actual definition of the configuration here. But well.

Reply:
I followed Analytics Read structure, and asked Louis Bourdages also. `It's a JSON` :|

Reply:
Well, if the lack of precision comes from the API, we can question it (you did), but that's pretty much it.
Loos OK to me then 👍